docker-compose.yml

```diff
@@ -1,11 +1,11 @@
-# dont need this anymore, but useful for test deployment
-# requires separate service providing postgres database
 services:
   pygeoapi:
     build: .
     container_name: reference-geoconnex-us
     ports:
       - 5000:80
+    volumes:
+      - ./pygeoapi.config.yml:/pygeoapi/local.config.yml
```

Comment:
I feel it is easier to iterate on by having it as a volume instead of copied in.